geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/Command.java

```diff
@@ -107,6 +109,24 @@ public RedisCommandType getCommandType() {
     return this.commandType;
   }
 
+  /**
+   * Convenience method to get the byte array representation of the key in a Redis command, always
+   * at the second position in the sent command array
+   *
+   * @return Returns the second element in the parsed command list, which is always the key for
+   *         commands indicating a key
+   */
+  public byte[] getBytesKey() {
+    if (this.commandElems.size() > 1) {
+      if (this.bytes == null) {
+        this.bytes = this.commandElems.get(1);
+      }
+      return this.bytes;
```

Comment:
Thanks for adding this method!  I realize the `bytes` variable was here beforehand, but could we rename it to something more descriptive (`keyBytes` and `keyString` for `key`) or something?

Also, it looks like the `setAsyncStartTime` and `getAsyncStartTime` methods in this class aren't used.  Not sure what they were used for previously.  I'd understand if you didn't want to do either of these things in this PR though.

Reply:
Done!